### DISCUSSION TITLE
Announcement on Discord?

### DISCUSSION THREAD

**Discussion by rakshit087** [nahuelwexd/vala-website#141] [General]
Since we have moved the website to vala-org, shall me make an announcement on the discord?
It will help connect more devs.

**Comment by colinkiama** [comment]
No need. It has already been shared on Discord :D
Colin
Who am I? <https://www.colinkiama.com>

**Reply by rakshit087** [reply]
I was thinking of sharing about it on the Vala Discord's announcement channel.

**Comment by colinkiama** [comment]
Oh ok. That's no problem.
Colin
Who am I? <https://www.colinkiama.com>
